pwa/src/components/footer/Footer.css

```diff
@@ -0,0 +1,18 @@
+.utrecht-page-footer {
+    display: flex;
+}
+
+.utrecht-page-footer > .utrecht-unordered-list {
+    flex: 1;
+}
+
+.utrecht-page-footer .utrecht-heading-4 {
+    color: white;
+}
+
+.utrecht-page-footer .utrecht-unordered-list__item {
+    --utrecht-icon-size: 20px;
+    display: flex;
+    align-items: center;
+    gap: 5px;
```

Comment:
The `gap` property unfortunately is not yet supported by all major browsers. Therefore, we have to add this margin in another way.

A possible solution would be:

```
.utrecht-page-footer .utrecht-unordered-list__item > svg {
  padding-right: var(--utrecht-space-inline-2xs);
}
```

Note that the hardcoded `5px` is also refactored to a design token.